examples/asset/asset_loading.rs

```diff
@@ -16,7 +16,8 @@ fn setup(
     mut materials: ResMut<Assets<StandardMaterial>>,
 ) {
     // By default AssetServer will load assets from inside the "assets" folder
-    // For example, the next line will load "assets/models/cube/cube.gltf#Mesh0/Primitive0"
+    // For example, the next line will load "./assets/models/cube/cube.gltf#Mesh0/Primitive0"
+    // You can set the Root by setting the enviroment variable "CARGO_MANIFEST_DIR" to the new root
```

Comment:
Can you expand these instructions a bit more? What makes a valid root, should I be setting a Windows environment variable or Rust environment variable, how do I go about doing that?

I know, but remember that we have users across all levels of technical backgrounds. This sort of configuration challenge can be particularly frustrating to debug on your own, since no error message appears that you can Google <3

Reply:
Good Point will add also add it to AssetServer::load

I have never heard about a Rust environment variable, and googling only reveals how to read an environment variable.

Reply:
Ah, this was in reference to:
> You can set the Root by setting the enviroment variable "CARGO_MANIFEST_DIR" to the new root

You need to change the cargo environment variables as described [here](https://doc.rust-lang.org/cargo/reference/environment-variables.html) for this fix to work. But on Windows, you can also access your system environment variables [like so](https://www.architectryan.com/2018/08/31/how-to-change-environment-variables-on-windows-10/), which is used for e.g. changing your PATH and is fully global to your system / user account.

Newer users may end up Googling the wrong thing and get hopelessly lost.

P.S. I noticed a typo on "enviroment"...